ocamltest/builtin_actions.ml

```diff
@@ -217,6 +232,19 @@ let use_runtime backend ocamlsrcdir = match backend with
     "-use-runtime " ^ ocamlrun
   | _ -> ""
 
+let runtime_variant_flags backend ocamlsrcdir =
+  let variant = runtime_variant() in
+  if variant=Normal then ""
+  else begin
+    let variant_str = if variant=Debug then "d" else "i" in
```

Comment:
I find this line a bit frustrating because it implicitly assumes that all non-`Debug` cases are `Instrumented`. What if we add other variants in the future? Would it be possible to use an exhaustive pattern matching here, and use `""` in the Normal case? (In fact, doesn't using `""` in the Normal case always work, even if you are passing a useless `-runtime-variant ""` setting?)